src/components/fb/utils.test.ts

```diff
@@ -196,4 +196,124 @@ describe('utils', () => {
       })
     })
   })
+
+  describe('time', () => {
+    const now = Date.UTC(2009, 0, 3)
+
+    const oneWeek = Date.UTC(1970, 0, 8)
+    const oneDay = Date.UTC(1970, 0, 2)
+
+    const oneDayFromNow = now + oneDay
+    const oneWeekFromNow = now + oneWeek
+    const fourWeeksFromNow = now + 4 * oneWeek
+    const oneMonthFromNow = Date.UTC(2009, 1, 3)
+    const twoMonthsFromNow = Date.UTC(2009, 2, 3)
```

Comment:
```suggestion
    const oneMonthFromNow = now + Date.UTC(1970, 1)
    const twoMonthsFromNow = now + Date.UTC(1970, 2)
```

NIT: Would need to adapt if `now` ever changed.. (which it never will, but.. yeah)..